mkdocs.yml

```diff
@@ -12,9 +12,9 @@ nav:
     - Building Environment: building/environment.md
     - Building Containers: building/containers.md
     - Building non CO-RE: building/nocore-ebpf.md
+    - Linux Headers: building/headers.md
```

Comment:
This should be in the Installation part, and not here

Reply:
Okey. Reopening #1562 to put it in the right place.

Reply:
I assumed headers.md was moved to building/ directory intentionally in #1351 but I guess I was wrong. 
I can create a new PR to address this issue as well

Reply:
Please do so @kaitoii11 And sorry for confusing you.

Reply:
Wait, actually I'm not sure we need it in either places anymore as it is already included in the prerequistes part